report/3.Theory/Dependability.tex

```diff
@@ -0,0 +1,45 @@
+\section{Dependability}
+Dependability in a real-time system is divided into two fields: hardware and software. 
```

Comment:
I like categories maybe instead of fields, but actually, fields is fine.

Reply:
categories is neato